reconcile/queries.py

```diff
@@ -191,6 +191,19 @@ def get_aws_accounts():
       service
       pod
     }
+    peering {
+      vpc_id
+      connections {
```

Comment:
Add a connection name to give the peering connection?  I know you generate one, but it's vpc IDs which may not be easy to recognize.  If not a name, then maybe prefix?

Reply:
i think that in general we may want to support an array here in the future.

i agree that we need a connection name. please allow me to save it for a future PR.